control/cli.py

```diff
@@ -187,12 +187,17 @@ def delete_bdev(self, args):
         argument("-n", "--subnqn", help="Subsystem NQN", required=True),
         argument("-s", "--serial", help="Serial number", required=False),
         argument("-m", "--max-namespaces", help="Maximum number of namespaces", type=int, default=0, required=False),
+        argument("-a", "--ana_reporting", help="Enable ANA reporting", type=bool, default=False, required=False),
```

Comment:
I think you should use "-" and not "_" to be consistent with other parameter names. That is "--ana-reporting" and "--enable-ha".